recipes-bsp/atf/atf-juno_git.bb

```diff
@@ -29,19 +33,27 @@ do_compile[depends] += "u-boot-juno:do_deploy"
 do_compile() {
     oe_runmake \
       CROSS_COMPILE=${TARGET_PREFIX} \
-      all \
-      fip \
-      PLAT=${COMPATIBLE_MACHINE} \
-      SPD=none \
       SCP_BL2=${WORKDIR}/juno-oe-uboot/SOFTWARE/scp_bl2.bin \
-      BL33=${DEPLOY_DIR_IMAGE}/u-boot.bin
+      BL32=${DEPLOY_DIR_IMAGE}/optee/tee-header_v2-juno-3ff350a1.bin \
```

Comment:
maybe use wildcard to avoid to hardcode 3ff350a1

Reply:
I was wondering if we can drop the gitrev in the optee do_deploy instead.

Reply:
works for me too. as long as we get rid of this hardcoded value and avoid to update this line in the future.

Reply:
https://review.linaro.org/22612